src/builddataset.py

```diff
@@ -13,45 +13,38 @@ def ingest(self, filepath):
             print("File path {} does not exist. Exiting...".format(filepath))
             sys.exit()
 
-
         analyzer = Sentiment("apikey.json")
-        inputFile = open(filepath, "r")
-        while True:
-            info = inputFile.readline().strip("\n")
-
-            # if line is empty, end of file is reached
-            if not info:
-                break
 
-            tweeterInfo = info.split(",")
-            tweeterHandle, tweeterLabel = tweeterInfo[0], tweeterInfo[1]
+        with open(filepath, "r") as f:
+            for info in f:
```

Comment:
I'm a bit confused here, I expected something like we read something like:
```
handle,party,number
```
and then read a bunch of tweets that are prefixed by a number. And then we repeat. It doesn't look like we're doing that?

Reply:
We are because the int call will fail because it will try to do int(handle), which will then go into the except clause to get the label

Reply:
buhr would be proud of you, using exceptions as control flow